test/test_linalg.py

```diff
@@ -7453,6 +7454,54 @@ def test_preferred_linalg_library(self):
         self.assertEqual(out_ref, out1.cpu())
         self.assertEqual(out1, out2)
 
+    @onlyCUDA
+    @toleranceOverride({torch.float16: tol(atol=1e-1, rtol=1e-1),
+                        torch.bfloat16: tol(atol=1e-1, rtol=1e-1),
+                        torch.float32: tol(atol=1e-1, rtol=1e-1)})
+    @dtypes(torch.float16, torch.bfloat16, torch.float32)
+    @parametrize("size", [100, 1000, 10000])
+    def test_cublas_addmm(self, size: int, dtype: torch.dtype, device):
+        #
+        # Check for catastrophic cuBLAS inaccuracy by measuring the deviation between
+        # results from the CUDA invocation of torch.addmm and the CPU invocation
+        # (which does not use CUDA backend).
+        #
+        # Get dims
+        n, m, p = (size + 1, size, size + 2)
+        # Make random tensors on CPU
+        torch.manual_seed(123456789)
```

Comment:
Why is this needed?

Reply:
For reproducibility of specific failed tests, and to enable comparison between dtypes.  However, we might also want to sample across different random matrices for each dtype/size combination.  In this case, it would be better to run each dtype/size combination several times, and having the same random test matrices matters less.

Reply:
I'm pretty sure this is already done at a test suit level.